USING-SNAPSHOT-ARTIFACTS.md

```diff
@@ -0,0 +1,47 @@
++# How to use Eclipse JKube snapshot artifacts?
+
+- +Artifacts are hosted at [JKube's Sonatype Snapshot repository](https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/jkube/)
+- +Our [Jenkins Snapshot release pipeline](https://ci.eclipse.org/jkube/job/ReleasePipelines/job/ReleaseSnapshots/) updates SNAPSHOT artifacts every night.
+-
+- +## Using SNAPSHOTs in Maven Project
+  +In order to use these artifacts, update your `pom.xml` with these:
+- +```xml +<pluginRepositories>
+- <pluginRepository>
+- <id>oss.sonatype.org</id>
+- <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+- <snapshots>
+-      <enabled>true</enabled>
```

Comment:
ah, sorry for not being clear. You're supposed to not add preceding `-`/`+` signs. They were added by `git diff` output.

Reply:
You can view preview of your work here https://github.com/Prafful-Rastogi/jkube/blob/pr/issue2254/USING-SNAPSHOT-ARTIFACTS.md

Reply:
Hi @rohanKanojia thats fine will make changes and let you know.